.ci/linux-steps.yaml

```diff
@@ -23,7 +23,8 @@ steps:
 
     git clone --depth 1 https://github.com/mlpack/jenkins-conf.git conf
 
-    sudo apt-get install -y --allow-unauthenticated libopenblas-dev g++ xz-utils
+    sudo apt-get install --yes --allow-unauthenticated \
+    	libopenblas-dev libarmadillo-dev libensmallen-dev libstv-dev libcereal-dev xz-utils
```

Comment:
```suggestion
    	libopenblas-dev libarmadillo-dev libensmallen-dev libstb-dev libcereal-dev xz-utils
```

Reply:
Jeebus.  Silly typo.  Good catch.  I will fold into the upcoming 'partial revert' commit.

Reply:
Done in 8a24a97